index.html

```diff
@@ -3798,60 +3795,59 @@ <h2>DID Resolution</h2>
 stream.
             </dd>
             <dt>
-did-document-metadata
+<dfn>didDocumentMetadata</dfn>
             </dt>
             <dd>
 If the resolution is successful, this MUST be a <a
 href="#metadata-structure">metadata structure</a>. This structure contains
 metadata about the <a>DID document</a> contained in the
-<code>did-document</code> or <code>did-document-stream</code>. This metadata
-typically does not change between invocations of the <code>resolve</code>
+<code>didDocument</code> property. This metadata
+typically does not change between invocations of the <code>resolveRepresentation</code>
```

Comment:
Why this change? `didDocumentMetadata` is returned both for the `didDocument` and for the `didDocumentStream`, returned by `resolve()` and `resolveRepresentation()` respectively.

Reply:
This is a bug, will fix.

Reply:
Fixed in 5d96fb87147d2921ce5079c00433f6bfcdd7335e.